tedana/decomposition/pca.py

```diff
@@ -53,41 +50,30 @@ def low_mem_pca(data):
 def tedpca(
     data_cat,
     data_oc,
-    combmode,
```

Comment:
Interesting that `combmode` and `t2sG` are no longer used. They were only useful for the kundu options and my best guess is that they stopped being used here when `metrics.collect.generate_metrics` was added. The one work is that those are doe within generate_metrics, but the option for `combmode` can no longer be externally set. I don't think this a problem or needs correction, but figured I'd highlight in case others see an issue.